File: Chapter 04/Python/section_4_4_anti_patterns/after_rich_domain_thin_controller/domain/models/customer.py
class Customer:
    """
    ARCHITECTURE NOTE: Not every domain model needs complex behavior. 
    Because the core business rules for this bounded context revolve 
    around the Order, this Customer class can remain a simple data 
    entity holding state.
    """
    def __init__(self, id: int = 0, type: str = "", email: str = ""):
        self.id = id
        self.type = type
        self.email = email

File: Chapter 04/JS/section_4_4_anti_patterns/after_rich_domain_thin_controller/domain/models/customer.js
/**
 * ARCHITECTURE NOTE: Not every domain model needs complex behavior. 
 * Because the core business rules for this bounded context revolve 
 * around the Order, this Customer class can remain a simple data 
 * entity holding state.
 */
class Customer {
    constructor(id = 0, type = "", email = "") {
        this.id = id;
        this.type = type;
        this.email = email;
    }
}
module.exports = Customer;